trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
```

Comment:
- [ ] I think if we know the length of the `correct_name` and then see if it > 10 🙃 
So the code can be:

```ruby
    @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name

```